unit-tests.gs

```diff
@@ -1,4 +1,6 @@
 /* global Logger Log SimplifiedSemanticVersion */
```

Comment:
Everything should be in a single line: `/* global Logger Log SimplifiedSemanticVersion log generateEmailNotification */`